Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -52,87 +52,95 @@ directives.
+
+## View properties
+
+* **cleanupIntervalStep** (_optional_; default: `10`; to disable use: `0`)<br/>
+  wait at least this many commits between removing unused files in the
+  ArangoSearch data directory
+  for the case where the consolidation policies merge segments often (i.e. a
+  lot of commit+consolidate). A lower value will cause a lot of disk space to
+  be wasted
+  for the case where the consolidation policies rarely merge segments (i.e.
+  few inserts/deletes). A higher value will impact performance without any
+  added benefits.
+  >With every "commit" or "consolidate" operation a new state of the view
+   internal data-structures is created on disk
+   old states/snapshots are released once there are no longer any users
+   remaining
+   however, the files for the released states/snapshots are left on disk, and
+   only removed by "cleanup" operation.
+
+* **commitIntervalMsec** (_optional_; default: `60000`; to disable use: `0`)<br/>
+  wait at least this many milliseconds between committing view data store
+  changes and making documents visible to queries
+  for the case where there are a lot of inserts/updates. A lower value will
+  cause the view not to account for them, (unlit commit), and memory usage
+  would continue to grow
+  for the case where there are a few inserts/updates. A higher value will
+  impact performance and waste disk space for each commit call without any
+  added benefits.
+  >For data retrieval ArangoSearch views follow the concept of
+   "eventually-consistent", i.e. eventually all the data in ArangoDB will be
+   matched by corresponding query expressions
+   the concept of ArangoSearch view "commit" operation is introduced to
+   control the upper-bound on the time until document addition/removals are
+   actually reflected by corresponding query expressions
+   once a "commit" operation is complete all documents added/removed prior to
+   the start of the "commit" operation will be reflected by queries invoked in
+   subsequent ArangoDB transactions, in-progress ArangoDB transactions will
+   still continue to return a repeatable-read state.
+
+* **consolidate** (_optional_; default: `{}`; to disable use: `null`)<br/>
+  the consolidation policy to apply for selecting data store segment merge
+  candidates.
+  >With each ArangoDB transaction that inserts documents one or more
+   ArangoSearch internal segments gets created
+   similarly for removed documents the segments that contain such documents
+   will have these documents marked as 'deleted'
+   over time this approach causes a lot of small and sparse segments to be
+   created
+   a "consolidation" operation selects one or more segments and copies all of
+   their valid documents into a single new segment, thereby allowing the
+   search algorithm to perform more optimally and for extra file handles to be
+   released once old segments are no longer used.
+
+  * **type** (_optional_; default `"bytes_accum"`)<br/>
```

Comment:
I like this structure better than the previous docs, but it is now harder to see that you have multiple formulas in a single `consolidate` object. Suggest to make that explicit.

Reply:
I'm not sure whether it should be done right now, but it could be addressed later.